app/jobs/deposit_collection_job.rb

```diff
@@ -59,14 +59,16 @@ def update_collection_record # rubocop:disable Metrics/AbcSize
     assign_contributors
   end
 
+  # rubocop:disable Metrics/AbcSize
   def perform_persist
     if !collection_form.persisted?
-      Sdr::Repository.register(cocina_object: mapped_cocina_object)
+      Sdr::Repository.register(cocina_object: mapped_cocina_object, user_name: Current.user.sunetid)
```

Comment:
How about this so you don't have to keep calling `Current.user.sunetid`?
```
def user_name
 Current.user.sunetid
end
```

Reply:
done